src/app/scripts/cac/map/cac-map-templates.js

```diff
@@ -258,36 +267,41 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
             '</div>',
             '{{#each data.legs}}',
                 '<div class="directions-leg" ',
-                    'data-lat="{{this.from.lat}}" data-lon="{{this.from.lon}}">',
-                    // transit step directions
+                'data-lat="{{this.from.lat}}" data-lon="{{this.from.lon}}">',
                     '{{#if this.transitLeg}}',
-                    '<div class="directions-step {{modeClass this.mode}}" ',
-                        'data-lat="{{ this.from.lat }}" data-lon="{{ this.from.lon }}">',
-                        '<div class="directions-instruction">Board {{this.agencyName}} {{this.route}} ',
-                        '{{this.headsign}}</div>',
-                        '<div class="directions-time">at {{datetime this.startTime}}</div>',
-                        '<div class="directions-distance">{{inMiles this.distance}} mi</div>',
-                    '</div>',
-                    '<div class="directions-step directions-step-disembark" ',
-                        'data-lat="{{ this.to.lat }}" data-lon="{{ this.to.lon }}">',
-                        '<div class="directions-instruction">Disembark <strong>',
-                            '{{this.to.name}}</strong></div>',
-                    '</div>',
-                    '{{else}}',
-                    // non-tranist step directions
-                    '{{#each steps}}',
-                        '<div class="directions-step {{directionClass this.relativeDirection ../this.mode}}" ',
-                            'data-lat="{{ lat }}" data-lon="{{ lon }}">',
-                            '<div class="directions-instruction">{{directionText}}</div>',
+                        // transit step directions
```

Comment:
Actually OTP doesn't return any steps for transit ("used for walking, biking and driving" in that comment isn't entirely explicit about what happens for other modes, but seemingly there's an implied "only").  I only changed the indentation of the transit block (which I should have done in a separate commit...) but yeah, this setup seems reasonable to me.